types/disqusjs/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false
```

Comment:
You should remove this rule exclusion.

There aren't any type parameters in the code, so it's not doing anything.

Reply:
Thank you for review, so can I just remove it, right?

Reply:
no, sorry, you still need the `extends` part =(

Reply:
That's okay, this will let me know that I should better understand tslint and the contribution process, that's cool😁